mne/viz/topomap.py

```diff
@@ -2806,3 +2807,206 @@ def plot_bridged_electrodes(info, bridged_idx, ed_matrix, title=None,
         cax = fig.colorbar(im)
         cax.set_label(r'Electrical Distance ($\mu$$V^2$)')
     return fig
+
+
+def plot_ch_adjacency(info, adjacency, ch_names, kind='2d', edit=False):
+    """Plot channel adjacency.
+
+    Parameters
+    ----------
+    info : instance of Info
+        Info object with channel locations.
+    adjacency : array
+        Array of channels x channels shape. Defines which channels are adjacent
+        to each other. Note that if you edit adjacencies
+        (via ``edit=True``), this array will be modified in place.
```

Comment:
requiring a numpy dense array in edit mode is I think an unecessary.
Yes inserting elements in a sparse array is inefficient (at least for CSR,  CSC formats) but it should not be critical for our size of problem. I would do something like this:

```
from scipy import sparse
import warnings

A = sparse.random(10, 10, density=0.1).tocsr()
with warnings.catch_warnings(record=True):
    A[0, 0] = 1
```

Reply:
Ok, I think I can rework the code for `csr_matrix`. So I undrestand that `coo_matrix` should never appear as the adjacency (I think this sparse format was used previously in mne). If so a check for either dense or csr (and otherwise error) would be useful.

Reply:
True, it would be nice to somehow solve this problem ... otherwise people will always write three lines:

```python
adj = find_ch_adjacency
adj = adj.to_array()  # <-- would be cool if we could get rid of this
plot_ch_adjacency(adj)
```

Reply:
plotting and editing sparse arrays is now possible, it was easier than I thought. :)